ios/RCCNavigationController.m

```diff
@@ -211,6 +211,8 @@ - (void)performAction:(NSString*)performAction actionParams:(NSDictionary*)actio
     
     RCCViewController *viewController = [[RCCViewController alloc] initWithComponent:component passProps:passProps navigatorStyle:navigatorStyle globalProps:nil bridge:bridge];
     
+    viewController.navigationItem.hidesBackButton = YES;
```

Comment:
Hm, so just so I'm understanding, by doing this, we make sure that the back button is hidden during the reset, and then once the JS context for the next view controller kicks in, it'll set the correct properties?

It's weird to me that the back button even shows in the first place when calling `setViewControllers`, since it should theoretically wipe out the whole nav stack...